lib/screens/ppg/heartRate.dart

```diff
@@ -230,9 +230,9 @@ class PPGView extends State<PPG> with SingleTickerProviderStateMixin {
       List _cameras = await availableCameras();
       _controller = CameraController(_cameras.first, ResolutionPreset.low);
       await _controller.initialize();
-      Future.delayed(Duration(milliseconds: 100)).then((onValue) {
-        _controller.flash(true);
-      });
+      // Future.delayed(Duration(milliseconds: 100)).then((onValue) {
```

Comment:
I mentioned it earlier as well, just delete the code if it's not required. Turning them into comments will only create garbadge.